### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by b01001011**
Issue: #28

Team agreed to have certain sets of rules on merging a pull request. 
That shall be recorded in the CONTRIBUTING file.
- [x] Create and update `Pull Request Merging Convention`
- [x] Test
- [x] Review
# Contributing to FunBot
## Pull Request Merging Convention
1. All the comments SHOULD to be addressed prior to merging.
2. At least 3 reviewers SHOULD approve
   - To express an approval, press 👍  on the decription.
3. If any participant requests specific member for a review (eg. @masunghoon), hold until the member reviews the PR.

Thanks! 

FunProg Team

### CODE REVIEW

**Comment by kwoolytech** [comment]
# Agreed?

Please press 👍  if you like this changes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Contributing to FunBot
+
+## Pull Request Merging Convention
+ 1. All the comments SHOULD to be addressed prior to merging.
+ 2. At least 3 reviewers SHOULD approve 
+  - Any one of participants shall post a comment with ```Agreed?``` header.
+
+  > # Agreed?
+ // anything. eg. Please press like if you agree with my code.
```
Comment (jwChung) [inline CONTRIBUTING.md]: 바로 밑 문장과 같이 `like` 를 `+1 emoji`로 표현하는 것은 어떨까요?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+ - To express an approval, press 👍 
+ 
+ 3. If any participant request specific member for a review (eg. @masunghoon) and hold until the member reviews the PR.
```
Comment (jwChung) [inline CONTRIBUTING.md]: 번데기 앞에서 주름 잡는 격인데, 저는 왜 아래 문장으로 쓰고 싶죠 -_-;;; 

If any participant **requests** specific member for a review (eg. @masunghoon), **hold** until the member reviews the PR.

**Comment by jwChung** [comment]
질문을 두개 남겨서 -1로 표시했습니다. 궁금한 점이 해소되면 +1로 변경하겠습니다.

File: CONTRIBUTING.md (hunk 2 above)
Comment (b01001011) [inline CONTRIBUTING.md]: Correct! Good call.

**Comment by b01001011** [comment]
Team, we have decided not to use `Agreed?` comment. Instead, we can put 👍 on the description. Be the first "Liker" for this PR. :)

**Comment by jwChung** [comment]
@funprog/core 이 PR 역시 한 분 더 리뷰해주시면 좋겠네요.

**Comment by jwChung** [comment]
+1 세 분 이상해 주셨서 병합했습니다~

**Pull request merged**